src/mixins/BeforeRouteLeave.js

```diff
@@ -0,0 +1,18 @@
+export default {
+    beforeRouteLeave(to, from, next) {
+        if (this.canLeave) {
+            next()
+            return;
+        }
+
+        const answer = window.confirm(
+            this.__('Do you really want to leave? You have unsaved changes!')
```

Comment:
> Do you really want to leave? You have unsaved changes.

Reply:
Fixed in https://github.com/noahlocke/nova-js/commit/1757ba07a13bb80de95d29eddc545f01ee8e7c75